src/linter/manifestJson/ManifestLinter.ts

```diff
@@ -89,16 +120,49 @@ export default class ManifestLinter {
 		}
 
 		// Detect deprecated view type in routing.config:
-		if (routing?.config && routing.config.viewType && deprecatedViewTypes.includes(routing.config.viewType)) {
+		const viewType = routing?.config?.viewType as string ?? routing?.config?.type;
```

Comment:
Isn't `viewType` and `type` something completely different?
https://github.com/SAP/ui5-manifest/blob/4e79c6f80d21cce52c52394877b61384c53d3ed4/schema.json#L8206-L8216
vs.
https://github.com/SAP/ui5-manifest/blob/4e79c6f80d21cce52c52394877b61384c53d3ed4/schema.json#L8157-L8164

Reply:
Yes and No 😅 

`type` used to be what `viewType` now represents. With the introduction of `viewType`, the `type` property started to have the values `View` and `Component`. But there are many places where `type` is still used to handle `XML`, `JS`, etc.

That's why we check if `viewType` is present first and then fallback to `type` here

Reply:
I'm not aware of such a change. The only place where I can see `type` is used for a view type is within the `sap.ui5/rootView`. Can you link to the relevant part in the schema?

Reply:
As commented [here](https://github.com/UI5/linter/pull/780#discussion_r2245865507), I could not find any evidence that there ever was a renaming of those properties in the "sap.ui5"/"routing".

Even in 1.38, the `viewType` was already named `viewType`:
https://ui5.sap.com/1.38.64/#docs/guide/902313063d6f45aeaa3388cc4c13c34e.html